changelogs/fragments/attachment_upload_module.yaml

```diff
@@ -0,0 +1,3 @@
+---
+major_changes:
```

Comment:
It is not breaking API, this result in change of minor version (2.1.0 -> 2.2.0).
```suggestion
minor_changes:
```

Reply:
Although major_changes by definition is not a breaking change, you are still right - from the documentation for major_changes https://docs.ansible.com/ansible/latest/community/development_process.html:  "SHOULD NOT include individual module or plugin changes. MUST include non-breaking changes that impact all or most of a collection".
I updated it to minor_changes.